content/_data/events/scale15x.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Scale 15x"
+date: "2017-03-02 thru 03-05T09:00:00"
```

Comment:
I had this same thought where I would only specify the first day of the conf but then after the first day the  event would fall off the calendar. My reason for putting the 'thru date' was so that the event stays visible on the calendar until at least its last day. But if this isn't feasible than i'll just stick w/ first date of event.